examples/generic-auth/README.md

```diff
@@ -0,0 +1,49 @@
+# Generic Auth example
+
+Showcase of doing authentication for Query, Mutation and Subscription operations.
+
+```bash
+yarn workspace example-generic-auth dev
+```
+
+Provide the authentication token `aaa` or `bbb` via the `x-authorization` header or the `x-authorization` query parameter.
```

Comment:
Should we omit `x-` and just use `Authorization` here? I know a lot of people frown at using `x-` these days for header prefixes.

Reply:
I choose this one because of the `Bearer blabla` sanitization overhead :D